domains/certificates/Query.API/API.UnitTests/GranularCertificateIssuer/EnergyMeasuredConsumerTest.cs

```diff
@@ -24,7 +24,7 @@ public class EnergyMeasuredConsumerTest
         Type: MeteringPointType.Production,
         Technology: new Technology(FuelCode: "F00000000", TechCode: "T010000"),
         MeteringPointOwner: "meteringPointOwner",
-        MeteringPointOnboarded: true);
+        MeteringPointOnboardedStartDate: DateTimeOffset.Now.AddDays(-1));
```

Comment:
We could add tests where 
- we are just before (e.g. 1 ms before) the issuance start date
- exactly on the issuance start date

This will require that we can control the time in EnergyMeasuredConsumer, but that can be done with an interface.
What do you think?